_rules/SC2-2-1+SC3-2-5-meta-refresh.md

```diff
@@ -1,77 +1,108 @@
 ---
-rule_id: SC2-2-1+SC3-2-5-meta-refresh
-name: Meta refresh and redirect is not used
-test_mode: automatic
-environment: DOM Structure
+name: Meta-refresh no delay
+description: |
+  This rule checks that the meta element is not used for delayed redirecting or refreshing.
 
 success_criterion:
-- 2.2.1 # Timing Adjustable (Level A)
-- 3.2.5 # Change on Request (Level AAA)
+- 2.2.1 # Timing Adjustable
+- 3.2.5 # Change on Request
 
-author:
+test_aspects:
+- DOM Tree
+
+authors:
+- Anne Thyme Nørregaard
 - Wilco Fiers
 ---
 
-## Description
+## Test procedure
 
-This test checks if meta element is not used for delayed redirecting or refreshing.
+### Applicability
 
-## Background
+The rule applies to any `content` attribute in a `meta` element that contains the `http-equiv` attribute with value `"refresh"` (case-insensitive).
 
-- [H76: Using meta refresh to create an instant client-side redirect](http://www.w3.org/TR/WCAG20-TECHS/H76.html)
-- [F40: Failure of Success Criterion 2.2.1 and 2.2.4 due to using meta redirect with a time limit](http://www.w3.org/TR/WCAG20-TECHS/F40.html)
-- [F41: Failure of Success Criterion 2.2.1, 2.2.4, and 3.2.5 due to using meta refresh with a time-out](http://www.w3.org/TR/WCAG20-TECHS/F41.html)
+### Expectation
 
-## Assumptions
+The value before the first semicolon or comma (representing seconds) of the value of the `content` attribute is not a number greater than 0.
 
-- This test assumes no functionality was provided by the website for the user to adjust the timer.
+**Note**: Semi-colon or comma is optional in the `content` attribute. When none is present, the value of the attribute should be considered as a whole. See [meta-refresh](https://www.w3.org/TR/html51/document-metadata.html#statedef-http-equiv-refresh) for parsing instructions.
```

Comment:
I'd suggest linking to the non-versioned version of the HTML spec: https://www.w3.org/TR/html/document-metadata.html#statedef-http-equiv-refresh

Reply:
This is also not quite accurate. Per the spec, space characters can also be used for separating the time and URL of a refresh, so the following is perfectly valid:

```html
<head>
  <meta http-equiv="refresh" content="0 http://example.com" />
</head>
```

This, on the other hand, will cause a redirect after 5 seconds:

```html
<head>
  <meta http-equiv="refresh" content="0: http://example.com" />
  <meta http-equiv="refresh" content="5; http://example.com" />
</head>
```